### ISSUE TITLE
TTFF remains empty (beta 3.10.2)

### ISSUE THREAD

**Issue by spaceserver** [barbeau/gpstest#649]
TTFF remains empty after getting a gps-fix.
All other values are shown correctly.
platform: Nougat 7.0
API-Level: 24
Device: SM-G935F
Rom-Type: modded Stock Rom with custom kernel

**To Reproduce**
i would assume that reproducing requires testing with a similar setup.. :-/

**Expected behavior**
To show the TTFF

**Observed behavior**
Further explanation would just be overkill.

**App, Device and Android version:** 

What version of GPSTest did you encounter this on (see "Help->About")? What make and model device (e.g., Samsung Galaxy S10) did you encounter this on?  What Android version (e.g., Android 10.0) are you running?  Is it the stock version from the manufacturer or a custom ROM?  What version of Google Play Services is running on your device?

**Screenshots:** 

![Screenshot_20230618-022516](https://github.com/barbeau/gpstest/assets/10487290/a729447e-5832-45a8-ae76-0a652b57291e)

best regards & keep up your good work! :-)

**Comment by barbeau** [comment]
Sorry for the delayed response, somehow I missed this when it was first reported.

tl;dr - looks like this is an issue with the modded kernel. The GNSS time is flagged (highlighted in red) as invalid, so the firmware is likely corrupting the data from the GNSS chip. Unfortunately there isn't much I can do to fix this - I suggest you report it to the modding community for the kernel.
